server/api/projects/[id]/overview.ts

```diff
@@ -0,0 +1,134 @@
+import { and, eq, between, sql, desc } from "drizzle-orm";
+import {
+  getFeedbackCountOfProject,
+  feedbackCountByStatus,
+  feedbackCountByCategory,
+  countForStatus,
+} from "../../../db/query/analytics";
+import { getFeedbacks } from "../../../db/query/feedback";
+import { Feedback } from "../../../../lib/types/project";
+
+// /api/projects/1/overview
+
+// {
+//   stats: {
+//     total,
+//     countByStatus,
+//     countByCategory,
+//     open,
+//     closed,
+//   },
+//   feedbacks: []
+// }
+
+export default eventHandler(async (event) => {
+  const session = await requireUserSession(event);
+  const userId = session.user.id;
+
+  /*
+    Daily -> Hourly stats
+    Weekly -> Daywise stats
+    Monthly -> Weekly stats
+    Yearly -> Monthly stats
+  */
+
+  const { getProjectId, getDateRangeFilter } = useValidation(event);
+  const projectId = await getProjectId();
+  const { start, end } = await getDateRangeFilter();
```

Comment:
What do these start and end variables do?

Reply:
uh. i forgot to remove. :-) Will update the PR

Reply:
it should be in ISO format.
```
new Date().toISOString()
'2023-08-09T13:54:45.021Z'
```